code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -62,9 +64,27 @@
 	succeed_activate()
 
 	X.throw_at(A, RAV_CHARGEDISTANCE, RAV_CHARGESPEED, X)
+	activate_particles(TRUE, X.dir)
 
 	add_cooldown()
 
+/datum/action/xeno_action/activable/charge/proc/activate_particles(boolean = FALSE, direction)
```

Comment:
autodoc. Also why is that arg just called boolean? Make it something informative.

Reply:
I've never been good at naming anything in general. I'll call it toggle and hope that's fine.